### ISSUE TITLE
[Android] Empty choices Json Array causes Parsing to fail

### ISSUE THREAD

**Issue by somdas** [microsoft/AdaptiveCards#5569]
# Platform
Android

# Author or host

Microsoft Teams

# Version of SDK

2.4.0

# Details

When the choices Json Array is empty for type - "Input.ChoiceSet", then the parsing of the whole Adaptive card fails. On Desktop the Card is rendered properly without any choices (in drop down). Same should happen on Android. Here is an example (part of the JSON)- 

```
 {
         "choices":[
            
         ],
         "isMultiSelect":false,
         "style":null,
         "value":"${scheduleDisplay.selectedRome}",
         "isRequired":false,
         "id":"room",
         "spacing":"None",
         "separator":false,
         "type":"Input.ChoiceSet"
 }
```
When such JSON is present we get the Exception as - java.io.IOException: "Could not parse required key: choices. It was not found"

**Comment by paulcam206** [comment]
I believe we're tracking this work in #5513...

**Comment by jonmill** [comment]
Fixed by PR #5613 and dupe of bug #5513
